pkg/agent/agent.go

```diff
@@ -428,12 +428,16 @@ func (i *Initializer) setupDefaultTunnelInterface(tunnelPortName string) error {
 
```

Comment:
Or maybe check localIPin line 411, if NoteRouteController is not the right place.

Reply:
Thanks Jianjun. I checked the localIP change in line 411.